doc/release-notes-1.1.0.rst

```diff
@@ -230,9 +230,9 @@ BSD library
 Board support
 -------------
 
-* Thingy:91 (``nrf9160_pca20035``):
+* Thingy:91 (``thingy91_nrf9160``):
 
-  *  Removed support for earlier hardware versions of ``nrf52840_pca20035`` and ``nrf9160_pca20035``.
+  *  Removed support for earlier hardware versions of ``thingy91_nrf52840`` and ``thingy91_nrf9160``.
```

Comment:
Do we need to "retroactively" change release notes?

Reply:
Agree. Victim of search and replace.

Reply:
Correct, this shouldn't be updated (or only if there is a reference that breaks, but I don't think that will be the case for the Thingy).